phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixEmbeddedDriver.java

```diff
@@ -91,8 +89,15 @@
     protected ReadOnlyProps getDefaultProps() {
         return DEFAULT_PROPS;
     }
-    
-    abstract public QueryServices getQueryServices() throws SQLException;
+
+    /**
+     *  get query services
+     * @param info properties info
+     * @return query services
+     * @throws SQLException if failed to get query service.
+     */
+    public abstract QueryServices getQueryServices(Properties info)
```

Comment:
I see. You made an interface changes. I would suggest keep the original method with depreciation tag and remove it after two release.

Reply:
> I see. You made an interface changes. I would suggest keep the original method with depreciation tag and remove it after two release.
thanks , this is a good idea, I will do..